tools/README.md

```diff
@@ -41,14 +41,17 @@ Usage sof-logger <option(s)> <file(s)>
 -o out_file		Specify the output file, instead of default stdout
 -t			Get traces from "/sys/kernel/debug/sof/trace", instead
 			of the default "/sys/kernel/debug/sof/etrace"
+-u baud			Input data from a UART
+-r			Less formatted output for chained log processors
 -p			Get traces from stdin, instead of the default
 			"/sys/kernel/debug/sof/etrace"
 -c			Set timestamp clock in MHz
 -e			Enable checking firmware version with default verification file
 			"/sys/kernel/debug/sof/fw_version"
 -v ver_file		Enable checking firmware version with ver_file file,
 			instead of default: "/sys/kernel/debug/sof/fw_version"
--s			Take a snapshot of state
+-s state_name		Take a snapshot of state. Save the debugfs entries in
+			state_name.*.txt.
```

Comment:
what happens when you have a snapshot, what do you do with it?
I never understood this option.

Reply:
@plbossart I imagine this may be useful for doing a quick diff between two such snapshots taken at different point in time in order to check what changed. I see this options has quite a long history actually as it comes from the initial soft rmbox tool https://github.com/thesofproject/soft/commit/74512df085f2b3788247404e7167f62ed09f1f2f

Reply:
Right, and my take when I see an historical 'feature' not being used by anyone is to ask questions. at my age ridicule isn't going to kill me :-)